### PR TITLE
Added documentation for criteria summary stats

### PR DESCRIPTION
**Pull request by anoverga17**
## Motivation
As part of pull request [#6100](https://github.com/MarkUsProject/Markus/pull/6100), the summary stats display has been modified to add new criteria statistics. This pull request aims to update the page on the summary stats page to show and explain these new criteria statistics.

## Changes
- Added description of new criteria graph
- Added description of new criteria table and what is shown
- Added headers for each section of the summary statistics page
- Updated relative information of where to find page components to account for new criteria stats section

### CODE REVIEW

File: Instructor-Guide--Assignments--Summary.md [hunk 1]
```diff
@@ -43,11 +45,32 @@ To the right of the grade distribution chart is a set of statistics that give a
...
 > :spiral_notepad: **Note:** In brackets, next to each of the statistics that are shown as a fraction, is the same statistic fraction but displayed as a percentage instead.
 
-Finally, at the bottom of the view, below the grade distribution chart, you can see another distribution of grades for an assignment. What differentiates this grade distribution graph from the one above however, is that each bar from the graph above is broken up in order to also show the distribution of marks given by each grader.
+#### Criteria Distribution
+
+Below the grade distribution chart, you can further analyze the distribution of grades using the criteria distribution graph. This graph shows the distribution of marks given for each associated criterion.
+
+![Criterion Grade Distribution Graph](images/criteria-summary-stats-grade-distribution.png)
+
+Each criterion corresponds to a colour shown in the labels just above the graph. By default, all grade data for criterions are hidden. In order to reveal the grade distribution for a specific criterion(s), simply click on the labels of all the criterion you wish to view and compare.
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply Instructor-Guide--Assignments--Summary.md:54]: `of all the criteria you wish to view and compare`

File: Instructor-Guide--Assignments--Summary.md [hunk 2]
```diff
@@ -43,11 +45,32 @@ To the right of the grade distribution chart is a set of statistics that give a
...
+
+To help you get a quick overview of student performance for each criteria, next to the criteria distribution graph is a table that shows each criterion along with the average grade received for that criterion. Clicking on the drop down arrow next to each criterion will reveal additional summary statistics for that particular criterion.
+
+![Criterion Grade Distribution Table](images/criteria-summary-stats-table.png)
+
+The additional criterion statistics currently shown are:
+
+- **Average:** The average point grade of submissions for the given criterion (including submissions which recieved a zero) out of the maximum possible mark for that criterion.
```
Comment (mishaschwartz) [inline Instructor-Guide--Assignments--Summary.md:62]: `the average point grade of submissions` doesn't sound right to my ear... can we change it to just `the average grade`

Comment (mishaschwartz) [reply Instructor-Guide--Assignments--Summary.md:62]: Do we really have to say `out of the maximum possible mark for that criterion`? That seems to be implied by both the pictures and the way that grades work on MarkUs and elsewhere.

Can we rephrase this whole thing to:

- **Average:** The average grade received for the given criterion
- **Median:** The median grade received for the given criterion
- etc.

And then at the bottom you can just write once: 

`All statistics include submissions which received a zero for the given criterion`

Comment (mishaschwartz) [reply Instructor-Guide--Assignments--Summary.md:62]: Can you also clarify whether this includes incomplete submissions and/or submissions that have not been released to students yet.

File: Instructor-Guide--Assignments--Summary.md [hunk 3]
```diff
@@ -43,11 +45,32 @@ To the right of the grade distribution chart is a set of statistics that give a
...
+- **Number of zeros:** How many graded submissions recieved a grade of zero on this particular criterion out of the expected total number of submissions.
+
+#### Grader Distribution
+
+Finally, at the bottom of the view, below the criteria distribution chart, you can see another distribution of grades for an assignment. What differentiates this grade distribution chart from the normal chart at the top of the page however, is that this chart breaks down the normal grade distribution graph in order to also show the distribution of marks given by each grader.
 
 ![Assignment Grader Distribution Graph](images/summary-stats-grader-distribution.png)
 
-Each grader corresponds to a colour shown in the labels just above the graph. Each label also shows how many submissions each grader has completed marking out of the number they were assigned.
+As with the criterion distribution, each grader corresponds to a colour shown in the labels just above the graph. Each label also shows how many submissions each grader has completed marking out of the number they were assigned.
```
Comment (mishaschwartz) [inline Instructor-Guide--Assignments--Summary.md:73]: You use `criteria distribution` elsewhere. Please change this one to be consistent with the others

File: Instructor-Guide--Assignments--Summary.md (hunk 2 above)
Comment (anoverga17) [reply Instructor-Guide--Assignments--Summary.md:62]: > Do we really have to say `out of the maximum possible mark for that criterion`? That seems to be implied by both the pictures and the way that grades work on MarkUs and elsewhere.
> 

Not really. This was worded to be consistent with the grade distribution documentation above and to be explicitly clear what the individual fractions shown meant. I'll change it though.

File: Instructor-Guide--Assignments--Summary.md [hunk 4]
```diff
@@ -43,11 +45,34 @@ To the right of the grade distribution chart is a set of statistics that give a
...
 > :spiral_notepad: **Note:** In brackets, next to each of the statistics that are shown as a fraction, is the same statistic fraction but displayed as a percentage instead.
 
-Finally, at the bottom of the view, below the grade distribution chart, you can see another distribution of grades for an assignment. What differentiates this grade distribution graph from the one above however, is that each bar from the graph above is broken up in order to also show the distribution of marks given by each grader.
+#### Criteria Distribution
+
+Below the grade distribution chart, you can further analyze the distribution of grades using the criteria distribution graph. This graph shows the distribution of marks given for each associated criterion.
+
+![Criterion Grade Distribution Graph](images/criteria-summary-stats-grade-distribution.png)
+
+Each criterion corresponds to a colour shown in the labels just above the graph. By default, all grade data for critera are hidden. In order to reveal the grade distribution for a specific criterion, simply click on the labels of all the criteria you wish to view and compare.
```
Comment (mishaschwartz) [inline Instructor-Guide--Assignments--Summary.md:54]: `all grade data for criteria` (missing an "i" in criteria)

**Review by mishaschwartz — APPROVED** [review]
Just the one tiny typo and then I'll pull it in. Looks great though

**Pull request merged**